tox.ini

```diff
@@ -1,4 +1,4 @@
-# Tox (http://tox.testrun.org/) is a tool for running tests
+# Tox (https://tox.testrun.org/) is a tool for running tests
```

Comment:
The subdomain link is dead and the organizational domain seems to be something else. Maybe this should point to https://tox.readthedocs.io/en/latest/ ?

Reply:
Yes, I think so